.github/workflows/pr.yml

```diff
@@ -18,6 +18,7 @@ jobs:
             { version: '2.12.12' },
             { version: '2.12.11' },
             { version: '2.12.10' },
+            { version: '2.13.5' },
             { version: '2.13.4' },
             { version: '2.13.3' },
             { version: '2.13.2' },
```

Comment:
So I think we need to actually remove all of these except for the actual version of Scala that sbt is running on here. I don't think having this matrix actually makes any sense since the by changing these here we aren't actually changing the versions in the scripted tests. For now, just remove them all except 2.12.13.

Reply:
Not sure about this, as you force sbt to run with 2.13.x. As this has been like this before I would probably keep it.

Reply:
On 2nd thought....I slimmed the list down.

Reply:
Well the change was made, and then tests have failed every since. From my understanding this is just changing the build version of the sbt plugin, which isn't what we want. If we are using the latest sbt, then it's 2.12.13. If we want to actual test the plugin for different Scala versions, then we would need to cross test in the scripted tests in the various `test` files. I don't believe this ever worked.

Reply:
Yeah "scripted" does not work, though you can manually run the tests from the sbt-test projects. Though I guess it is really not necessary. Only benefit I would have seen (of actually building the plugin for 2.13) is binary compatibility with scala 3. With those projects you never know when you have to update them again.